src/workers/notifications/communityPicksSucceeded.ts

```diff
@@ -0,0 +1,26 @@
+import { messageToJson } from '../worker';
+import { Post } from '../../entity';
+import { NotificationPostContext } from '../../notifications';
+import { NotificationWorker } from './worker';
+
+interface Data {
+  postId: string;
+  scoutId: string;
+}
+
+const worker: NotificationWorker = {
+  subscription: 'api.community-picks-succeeded-notification',
+  handler: async (message, con) => {
```

Comment:
In this example are you not afraid it adds a lot of overhead?
As this doesn't particularly do anything expect re-sending the data?

Code wise happy with how it looks, but wondering if executing directly in the first workers doesn't help with load?

Reply:
An alternative could be to add it directly to the code where we add new post I guess? Or have you thought about something else?

Reply:
Hmm more referring to the notification worker, that could execute directly, but see your thinking so let's leave this as is :)